components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/main/java/org/wso2/carbon/identity/notification/sender/tenant/config/NotificationSenderManagementConstants.java

```diff
@@ -0,0 +1,189 @@
+    public static final String INLINE = "inline";
+    public static final String EMAIL_INLINE_BODY = "{{body}}{{footer}}";
+    public static final String TO = "to";
+    public static final String ADAPTER_TYPE_KEY = "eventAdapterType";
+    public static final String ADAPTER_TYPE_EMAIL_VALUE = "email";
+    public static final String ADAPTER_TYPE_HTTP_VALUE = "http";
+    public static final String ADAPTER_PROPERTY = "property";
+    public static final String ADAPTER_PROPERTY_NAME = "name";
+    public static final String EMAIL_ADDRESS_PROPERTY = "email.address";
+    public static final String EMAIL_ADDRESS_VALUE = "{{send-to}}";
+    public static final String EMAIL_TYPE_PROPERTY = "email.type";
+    public static final String EMAIL_TYPE_VALUE = "{{content-type}}";
+    public static final String EMAIL_SUBJECT_PROPERTY = "email.subject";
+    public static final String EMAIL_SUBJECT_VALUE = "{{subject}}";
+    public static final String SMTP_PASSWORD_PROPERTY = "mail.smtp.password";
+    public static final String SMTP_FROM_PROPERTY = "mail.smtp.from";
+    public static final String SMTP_USER_PROPERTY = "mail.smtp.user";
+    public static final String SMTP_HOST_PROPERTY = "mail.smtp.host";
+    public static final String SMTP_PORT_PROPERTY = "mail.smtp.port";
+    public static final String HTTP_URL_PROPERTY = "http.url";
+    public static final String CONSTANT_HTTP_POST = "HttpPost";
+
+    public static final List<String> INTERNAL_PROPERTIES =
+            Collections.unmodifiableList(Arrays.asList(STREAM_NAME, STREAM_VERSION, PUBLISHER_TYPE_PROPERTY));
+    public static final List<String> PROPERTIES_TO_SKIP_AT_ADAPTER_CONFIG =
+            Collections.unmodifiableList(
+                    Arrays.asList(STREAM_NAME, STREAM_VERSION, PUBLISHER_TYPE_PROPERTY, INLINE_BODY_PROPERTY));
+
+    /**
+     * Enums for error messages.
+     */
+    public enum ErrorMessage {
+
+        // Client errors 600xx.
+        ERROR_CODE_PUBLISHER_NOT_EXISTS_IN_SUPER_TENANT("60001",
+                "No matching notification sender found in super tenant.",
+                "Can not find a notification sender named: %s, in carbon.super."),
+        ERROR_CODE_CONFLICT_PUBLISHER("60002", "Notification sender already exists.",
+                "There exists a notification sender: %s in the tenant."),
+        ERROR_CODE_SMS_PROVIDER_REQUIRED("60003", "Required attribute is missing",
+                "SMS provider is not defined for notification sender."),
+        ERROR_CODE_SMS_PAYLOAD_NOT_FOUND("60004", "SMS send API payload is not defined",
+                "SMS send API payload for provider: %s, is not defined in file or POST body properties."),
+        ERROR_CODE_SMS_PROVIDER_URL_REQUIRED("60005", "Required attribute is missing",
+                "SMS provider url is not defined for notification sender."),
+        ERROR_CODE_PUBLISHER_NOT_EXISTS("60006", "No notification sender found.",
+                "No notification sender found : %s"),
```

Comment:
All these are input validations.. why are these not consistent?

Reply:
These error codes are already defined in  https://github.com/wso2/identity-api-server/pull/368/files#diff-8ef58086d9a4911e6c1e10b51a8b6c80892035b11d7bb3d2859bb64784078789L114-L124.